tree/20_cfe_basics/cfengine/paths.cf

```diff
@@ -107,7 +107,6 @@ bundle common paths
       "path[lsattr]"        string => "/usr/bin/lsattr";
       "path[tar]"           string => "/bin/tar";
       "path[chkconfig]"     string => "/sbin/chkconfig";
-      "path[update_rcd]"    string => "/usr/sbin/update-rc.d";
```

Comment:
Removing this because it was an error, and has been removed from upstream cfengine library. It is defined as update_rc_d later in the file, on debian only.